src/app/profiles/store/profile.selectors.ts

```diff
@@ -0,0 +1,17 @@
+import { select } from '@angular-redux/store';
+import { RootState } from '../../store';
+import { Observable } from 'rxjs/Observable';
+
+export class ProfileSelectors {
+  @select(getProfileFormHome) static readonly home: Observable<string>;
+  @select(getProfileFormMembersNum) static readonly membersNum: Observable<string>;
+}
+
+export function getProfileFormHome(state: RootState) {
```

Comment:
You might want to consider looking for a way to decouple your feature state from the root state. In ngrx you can do this with StoreModule.forFeature & createFeatureSelector. However, I'm not sure exactly how to do this with ngRedux. However, this pattern is something you probably should explore since lazy loading a module might come into play at some point and affect this. It also could remove dependencies on "knowing" about the root state.